code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -13,6 +13,8 @@
 	actions_types = list(/datum/action/item_action/toggle_helmet_light)
 
 	//Species-specific stuff.
+	var/list/hide_tail_by_species = null

```

Comment:
Why do we need to redefine this for helmets of all things? Tails are on the body affected by the suit.